client/components/Pipeline.jsx

```diff
@@ -168,11 +173,18 @@ export default class Pipeline extends React.Component {
         style = styles.cardCancelled;
         break;
     }
+
+    let buildTitle = pipeline.name;
+
+    if (showBuildLabels) {
+      buildTitle = <div>{pipeline.name}<span style={styles.cardLabel}> : {pipeline.label}</span></div>;
+    }
+
     return (
       <Card style={style} containerStyle={styles.cardContainer}>
         <CardHeader
           className="buildtitle"
-          title={pipeline.name}
+          title={buildTitle}
```

Comment:
I would like this to be shown in the subtitle instead (where the status is shown). Since the label is related to the status. So instead of showing it like `pipeline.name : pipeline.label` in the title, show it like `pipeline.label : pipeline.status` in the subtitle. The label should have font-weight normal.